src/main/java/seedu/realodex/logic/parser/ParserUtil.java

```diff
@@ -257,7 +256,6 @@ public static ParserUtilResult<Family> parseFamilyReturnStored(String family) {
         if (!Family.isValidFamily(trimmedFamily)) {
             return new ParserUtilResult<>(Family.MESSAGE_CONSTRAINTS, new Family());
         }
-        assert Integer.parseInt(trimmedFamily) >= 1;
```

Comment:
nice spot! assert should not be used in this case since it depends on user input